test/functional/test_server.py

```diff
@@ -20,31 +20,37 @@
 
 
 def test_server():
```

Comment:
Could you rename this test to be more specific (to describe what it tests)? like "test_server_conforms_to_hosting_schema"

Reply:
Renamed to test_server_with_a_simple_app